src/cmd/cli/command/compose.go

```diff
@@ -45,6 +67,11 @@ func makeComposeUpCmd() *cobra.Command {
 
 			printPlaygroundPortalServiceURLs(deploy.Services)
 
+			var hasOnlyManagedResources = IsOnlyManagedResources(project)
+			if hasOnlyManagedResources {
+				term.Warn("Deploying project consisting of only managed services, please use the AWS console to monitor status")
+			}
```

Comment:
I'm inclined to try to be more specific in these warning messages. First, I think we should mention explicitly that defang cannot monitor status for managed services, and second I think we should mention the name of the managed services if we can.

Maybe we could write a new `ManagedResourceNames` func. Something like this:

```go
func ManagedResourceNames(project *compose.Project) (names []string) {
	for _, service := range project.Services {
		for k := range service.Extensions {
			if !slices.Contains(managed_resources, k) {
				names = append(names, service.Name)
				break
			}
		}
	}

	return names
}
```

It's not clear if we need the original `IsOnlyManagedResources` func elsewhere, but maybe we could replace it with a new `HasManagedResources` func which is written in terms of  `ManagedResourceNames`:
```go
func HasManagedResources(project *compose.Project) bool {
	return len(ManagedResourceNames(project)) > 0
}
```

Then we could use those names in the error message:
```suggestion
			managedResourceNames := ManagedResourceNames(project)
			hasManagedServices := len(managedResourceNames) > 0
			if hasManagedServices {
				term.Warn("Defang cannot monitor status of the following managed services: ", strings.Join(managedResourceNames, ", "), "Please use the AWS console directly.")
			}
```